hvplot/tests/testtransforms.py

```diff
@@ -19,6 +19,38 @@ def test_pandas_transform(self):
         )
         self.assertEqual((scatter.data['probability']).values, demo_df['probability'].values * 100)
 
+    def test_pandas_dim_size_alias(self):
+        df = pd.DataFrame({'x': np.linspace(0, 4, 5), 'y': np.linspace(1, 5, 5)})
+        dim_expr = hv.dim('y') * 10
+        scatter = df.hvplot.scatter('x', 'y', s=dim_expr)
+        plot_opts = scatter.opts.get().kwargs
+        assert repr(plot_opts.get('size')) == repr(dim_expr)
+
+    def test_pandas_dim_color_expression(self):
+        df = pd.DataFrame({'x': np.linspace(0, 4, 5), 'y': np.linspace(-2, 2, 5)})
+        dim_expr = hv.dim('y') * 10
+        scatter = df.hvplot.scatter('x', 'y', color=dim_expr)
+        plot_opts = scatter.opts.get().kwargs
+        assert repr(plot_opts.get('color')) == repr(dim_expr)
+        assert plot_opts.get('colorbar') is True
+
+    def test_pandas_color_alias_column(self):
+        df = pd.DataFrame({'x': np.linspace(0, 4, 5), 'y': np.linspace(1, 5, 5)})
+        scatter = df.hvplot.scatter('x', 'y', c='y')
+        plot_opts = scatter.opts.get().kwargs
+        assert repr(plot_opts.get('color')) == repr(hv.dim('y'))
+
+    # def test_pandas_dim_size_matplotlib_backend(self):
+    #     hv.extension('matplotlib')
```

Comment:
I commented out this test because of this line. I have no trust at all in the test suite that the extension is properly handled (reset). I'm planning to refactor the test suite soon, so when I get there I'll re-enable it.

Reply:
OK, thanks.